server/src/messaging/RabbitMQMessaging.js

```diff
@@ -0,0 +1,19 @@
+import { BrokerAsPromised as Broker } from 'rascal';
+import { config } from './config';
+
+const publishMessage = (async (payload, resultHandler) => {

```

Comment:
You can suppress these warnings by putting the following line at the top of the file:
```js
/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```
Now that we're mixing ts and js more and more in the codebase, we may want to [disable the rule for all js files](https://github.com/typescript-eslint/typescript-eslint/blob/v4.4.0/packages/eslint-plugin/docs/rules/explicit-module-boundary-types.md#configuring-in-a-mixed-jsts-codebase).